### DISCUSSION TITLE
Vue.js devtools in Chrome not working - Vue.js not detected

### DISCUSSION THREAD

**Discussion by joshuaohana** [quasarframework/quasar#10351] [General - Components / Directives / etc] [answered]
Vue.js devtools works on normal Vue apps just fine, but on my Quasar app it says "Vue.js not detected" when I click the icon

Chrome Version 92.0.4515.131
quasar@2.0.3
vue@3.1.5

and I've confirmed that the Vue.js devtools Chrome extension has "Allow access to file URLs" enabled

any ideas? It's very frustrating dealing without :-\

**Comment by pdanpdan** [comment, marked as answer]
Install the vue extension for vue3
